environment.yml

```diff
@@ -0,0 +1,17 @@
+usage: conda-env [-h] {create,export,list,remove,update,config} ...
+
+positional arguments:
+  {create,export,list,remove,update,config}
+    create              Create an environment based on an environment file
+    export              Export a given environment
+    list                List the Conda environments
+    remove              Remove an environment
+    update              Update the current environment based on environment
+                        file
+    config              Configure a conda environment
+
+optional arguments:
+  -h, --help            Show this help message and exit.
+
+conda commands available from other packages:
+  env
```

Comment:
This file doesn't look like a valid `environment.yml`, but rather like the accidental output of `conda env > environment.yml` (instead of maybe `conda list > environment.yml`).

Reply:
@imagejan Thanks!! Addressing this now.